### PR TITLE
Thumbnail Rotation

### PR DESCRIPTION
**Pull request by geoffmoss0**
It was annoying me so I fixed it, I think

### CODE REVIEW

**Comment by RamZallan** [comment]
@geoffmoss0 did you test this locally?

**Comment by geoffmoss0** [comment]
Tested the python locally, worked with images I got from gallery. Basically copy/pasted that code and made sure it worked when I manually gave it a file.

File: gallery/file_modules/__init__.py [hunk 1]
```diff
@@ -39,6 +39,17 @@ def generate_thumbnail(self):
         self.thumbnail_uuid = hash_file(self.file_path)
 
         with Image(filename=self.file_path) as img:
+
+			# read metadata to find orientation of image
+			# then rotate accordingly. 
+			rotation = img.metadata["exif:Orientation"]
```
Comment (RamZallan) [inline gallery/file_modules/__init__.py]: Can you fix the indentation? Looks like you're maybe using tab characters which aren't playing nice w/ the surrounding 4-space tabs.

Comment (RamZallan) [reply gallery/file_modules/__init__.py]: In addition to this, where are you finding `metadata`? Looks like we use exif dicts, with a `get_exif()` function.

Comment (geoffmoss0) [reply gallery/file_modules/__init__.py]: From what I remember and just looked up, metadata is just a read-only part of Image that contains the exif data. This implementation is just what I found about how to get exif data and it worked so I didn't question it.

Comment (geoffmoss0) [reply gallery/file_modules/__init__.py]: Also ew 4 space tabs

Comment (mxmeinhold) [reply gallery/file_modules/__init__.py]: Please see [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) for indentation.

Comment (geoffmoss0) [reply gallery/file_modules/__init__.py]: Can I easily edit change this without making a new PR? I cleaned it up locally

Comment (mxmeinhold) [reply gallery/file_modules/__init__.py]: Yeah just commit and push to your branch.

**Pull request closed without merging**